static/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -37,3 +37,68 @@ Once initialized in a project, DVC populates its installation directory
 - `.dvc/updater.lock` - a lock file for `.dvc/updater`.
 
 - `.dvc/lock` - a lock file for the whole dvc project.
+
+## Structure of cache directory
+
+The structure of directory depends if the data is stored in a file or is in a
```

Comment:
It's not the `structure of directory depends`, it's more like the way the data is stored in cache.